libs/workspace-manager/src/lib/workspace-manager/features/connect-workspace/connect-workspace.component.html

```diff
@@ -1 +1,12 @@
-<p>connect-workspace works!</p>
+<form
+  [formGroup]="form"
+  class="connect-workspace-form"
+  (submit)="connectWorkspace($event)"
```

Comment:
what happened to old good (ngSubmit)? And you won't need `preventDefault`

Reply:
fixed